index.js

```diff
@@ -15,3 +15,179 @@ hamburger.addEventListener('click', show);
+    listItem1: 'html',
+    listItem2: 'bootstrap',
+    listItem3: 'Ruby on rails',
+    image: 'images/Calender.png',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+
+  {
+    id: '4',
+    name: 'Professional Art',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    listItem1: 'html',
+    listItem2: 'bootstrap',
+    listItem3: 'Ruby on rails',
+    image: 'images/tree.png',
+    liveLink: '#',
+    sourceLink: 'https://github.com/Yerimah/My-portfolio.git',
+  },
+
+  {
+    id: '5',
+    name: 'Professional Art',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    listItem1: 'html',
+    listItem2: 'bootstrap',
+    listItem3: 'Ruby on rails',
+    image: 'images/portfolio.png',
+    liveLink: '#',
+    sourceLink: 'https://github.com/Yerimah/My-portfolio.git',
+  },
+
+  {
+    id: '6',
+    name: 'Professional Art',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    listItem1: 'html',
+    listItem2: 'bootstrap',
+    listItem3: 'Ruby on rails',
+    image: 'images/Calender.png',
+    liveLink: '#',
+    sourcelink: 'https://github.com/Yerimah/My-portfolio.git',
+
+  },
+];
+
+const projectButton = document.querySelectorAll('.card-btn');
+const windowContainer = document.createElement('div');
+const container = document.createElement('div');
+function closeIt() {
+  document.body.removeChild(windowContainer);
+  window.location.reload();
+}
+
+projectButton.forEach((el) => el.addEventListener('click', () => {
+  const displayedArray = cardWorks.filter((projectId) => projectId.id === el.getAttribute('id'));
+
+  for (let i = 0; i < displayedArray.length; i += 1) {
+    document.body.appendChild(windowContainer);
+    windowContainer.classList.add('popup-window');
+    windowContainer.appendChild(container);
+    container.classList.add('pop-up');
+    const img = document.createElement('img');
+    img.classList.add('pop-image');
+    img.setAttribute('src', displayedArray[i].image);
+
+    container.appendChild(img);
+
+    const title = document.createElement('h3');
+    title.classList.add('pop-title');
+    title.textContent = displayedArray[i].name;
+    container.appendChild(title);
+    const headContainer = document.createElement('div');
+    container.insertBefore(headContainer, img);
+    headContainer.className = 'headContainer';
+
+    const ul = document.createElement('ul');
+    ul.classList.add('pop-list');
+    container.appendChild(ul);
+    headContainer.append(title, ul);
+    const listItem1 = document.createElement('li');
+    listItem1.textContent = displayedArray[i].listItem1;
+    listItem1.classList.add('pop-cell');
```

Comment:
- [ ] Great job on making the requirements of the previous reviewer but I am sorry I can't approve this project because when you when making changes suggested by the previous reviewer, some part of the code is now broken, kindly I would recommend you to rework on the popup window because for now, this functionality is not working in this current branch `portfolio-popup-window`. check the screen video on how I am seeing your project

https://user-images.githubusercontent.com/65068771/147275771-e6c2da8c-bf2a-476c-9ec1-797d8eb58315.mp4

## HAPPY CODDING 😃  💻

Reply:
I have fixed this problem. the popup window is working well now

Reply:
I have made the changes and everything is working well now...I would like you to look at it once more
before I send it for review because I have just one review left

Reply:
Hello, @Yerimah  I have rechecked, everything is okay, I think you can resubmit it. Thanks